webtasks/cron_backend_mongodb.js

```diff
@@ -319,6 +330,71 @@ router.put('/:container/:name',
             .then(res.json.bind(res), next);
 });
 
+router.put('/:container/:name/state',
+    ensure('headers', ['host']),
```

Comment:
I don't think we care about the host header any more? The cluster host is explicitly configured in the token parameters.